src/hacbs/components/topology/factories/GraphErrorState.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { Alert, AlertActionCloseButton, Bullseye } from '@patternfly/react-core';
+import { OutlinedEyeSlashIcon } from '@patternfly/react-icons/dist/esm/icons/outlined-eye-slash-icon';
+import { global_palette_black_600 as grayColor } from '@patternfly/react-tokens/dist/js/global_palette_black_600';
+import cx from 'classnames';
+
+import './GraphErrorState.scss';
+
+type GraphErrorStateProps = {
+  errors: unknown[];
+  fullHeight?: boolean;
+};
+
+const GraphErrorState: React.FC<GraphErrorStateProps> = ({ errors, fullHeight }) => {
+  const [closedError, setClosedError] = React.useState<number[]>([]);
+
+  const uniqueErrors = React.useMemo(
+    () =>
+      errors.reduce((acc: string[], e: { message: string }): string[] => {
+        if (!acc.includes(e.message)) {
+          acc.push(e.message);
+        }
+        return acc;
+      }, []) as string[],
```

Comment:
nit: you should cast the declaration and not the result.
```suggestion
      }, [] as string[]),
```